### ISSUE TITLE
Left mouse button camera

### ISSUE THREAD

**Issue by Samidaro** [Drahsid/HybridCamera#6]
Hi,
could you please make it so the left mouse button moves only the camera like on standard mode. So we can move just the camera with left and eveything with right MB.
Thanks

**Comment by Drahsid** [comment]
This behavior is intended. You can partially get the behavior you want by setting "Force camera rotation to use mode" to "Standard", however, this will probably feel pretty awful.

**Comment by Samidaro** [comment]
yeah it's pretty unpredictable. Any way that could be improved upon?

**Comment by Samidaro** [comment]
hi, haven't played in a couple of months unfortunately. Will check when I next resubscribe. Thanks for your hard work tho!
